project-base/storefront/components/Blocks/Product/LastVisitedProducts/LastVisitedProductsContent.tsx

```diff
@@ -0,0 +1,33 @@
+import { ProductsSlider } from 'components/Blocks/Product/ProductsSlider';
+import { SkeletonModuleLastVisitedProducts } from 'components/Blocks/Skeleton/SkeletonModuleLastVisitedProducts';
+import { useProductsByCatnumsApi } from 'graphql/generated';
+import { GtmProductListNameType } from 'gtm/types/enums';
+
+type LastVisitedProductsProps = {
+    productIds: string[];
```

Comment:
Just a small detail, it's not IDs, but catalog numbers (`catnums`)

Reply:
For the sake of this component it was easier to use universal ID because it is just unique identificator for products in this component. How we are using those ids further (mapping them as catnums to API) doesn't matter. But if you see this as issue we can discuss this further.

Reply:
In my opinion it makes more sense to call it "catnums" as then you are defining the function interface in a way that it is easier for others to use it. If I have a function which will only work if I provide a catnum and will not work if I provide a UUID or ID, I would rather call the parameter "catnums" than something generic. It's about setting the expectations for the person who uses it, without him needing to analyze the component content. Think about it. If you don't agree, let's schedule a call or find a 3rd person to discuss this with.

Reply:
No need for further discussion. No problem, I don't have any strong preference as you do. So I implemented with `catnums`.